.gitlab-ci/stages/scan.yml

```diff
@@ -1,20 +1,22 @@
 #
 
-.scan_stage: &scan_stage
-  stage: "Vuln scanner"
-  dependencies: []
-  before_script:
-    - source .gitlab-ci/env.sh
-  except:
-    variables:
-      - $PRODUCTION
-
 Snyk docker scan:
-  <<: *scan_stage
+  stage: "Deploy"
```

Comment:
I wonder If we should move it to register stage ? since it needs register to complete ?